spec/support/concerns/child_minder_shared_examples.rb

```diff
@@ -1,7 +1,8 @@
 shared_examples 'a ChildMinder' do |resource_factory,
   valid_child_file_sym,
   invalid_child_file_sym,
-  child_folder_sym|
+  child_folder_sym,
+  is_trashable: false|
```

Comment:
The addition of this `is_trashable` parameter, and the resulting branch logic, has made the 'a ChildMinder' shared examples extremely difficult to read. I'm not sure what the best way to refactor this is, but I'm willing you pair with you to help figure it out.